http/jetty12/pom.xml

```diff
@@ -414,6 +422,21 @@
             <artifactId>jetty-session</artifactId>
             <version>${jetty.version}</version>
         </dependency>
+        <dependency>
+           <groupId>org.eclipse.jetty.websocket</groupId>
+           <artifactId>jetty-websocket-jetty-api</artifactId>
+           <version>${jetty.version}</version>
+        </dependency>
+        <dependency>
+           <groupId>org.eclipse.jetty.websocket</groupId>
+           <artifactId>jetty-websocket-jetty-server</artifactId>
+           <version>${jetty.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.eclipse.jetty.websocket</groupId>
+          <artifactId>jetty-websocket-jetty-client</artifactId>
+          <version>${jetty.version}</version>
+        </dependency>
         <dependency>
```

Comment:
I think we should not include the client and only server parts

Reply:
Done